test/util/Util.ts

```diff
@@ -17,6 +17,8 @@ const portNames = [
   'SparqlStorage',
   'Subdomains',
   'WebSocketsProtocol',
+
+  'unit-BaseHttpServerFactory',
```

Comment:
Newline and different naming convention?

Reply:
All the other ones there were integration tests so I wanted to differentiate. Not super important.

Reply:
Maybe just with comments in this file then?

```
// Integration
…
// Unit
…
```